torchtune/modules/transformer.py

```diff
@@ -206,6 +206,5 @@ def forward(self, tokens: Tensor, input_pos: Optional[Tensor] = None) -> Tensor:
         # shape: [b, s, d]
         h = self.norm(h)
 
-        # shape: [b, s, v]
```

Comment:
nit: I would still keep some version of shape comment in just cause it's generally useful for people. could just be `# shape: [b, s, out_dim]` or something like that